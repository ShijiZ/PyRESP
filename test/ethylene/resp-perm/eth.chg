All values are reported in atomic units
%FLAG TITLE
 molecule 1: ethylene

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
   1   0.0000000E+00   0.0000000E+00   1.2441670E+00
   2   0.0000000E+00   0.0000000E+00  -1.2441670E+00
   3   0.0000000E+00   1.7277940E+00   2.3159880E+00
   4   0.0000000E+00  -1.7277940E+00  -2.3159880E+00
   5   0.0000000E+00   1.7277940E+00  -2.3159880E+00
   6   0.0000000E+00  -1.7277940E+00   2.3159880E+00

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
   1          6          0    -2.4005038E-01
   2          6          1    -2.4005038E-01
   3          1          0     1.2052110E-01
   4          1          3     1.2052110E-01
   5          1          3     1.2052110E-01
   6          1          3     1.2052110E-01

%FLAG PERM DIP LOCAL: 3(I4,X5),I4,X2,E16.7
 dip.no   atm.no   ref.no   ivary      p(opt)
   1        1        2        0    -2.0286052E-01
   2        1        3        0     5.3659382E-02
   3        1        6        2     5.3659382E-02
   4        2        1        1    -2.0286052E-01
   5        2        4        2     5.3659382E-02
   6        2        5        2     5.3659382E-02
   7        3        1        0    -1.1791048E-01
   8        4        2        7    -1.1791048E-01
   9        5        2        7    -1.1791048E-01
  10        6        1        7    -1.1791048E-01

%FLAG PERM DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1   0.0000000E+00   0.0000000E+00   2.5943348E-01
   2   0.0000000E+00   0.0000000E+00  -2.5943348E-01
   3   0.0000000E+00   1.0019714E-01   6.2156367E-02
   4   0.0000000E+00  -1.0019714E-01  -6.2156367E-02
   5   0.0000000E+00   1.0019714E-01  -6.2156367E-02
   6   0.0000000E+00  -1.0019714E-01   6.2156367E-02

%FLAG IND DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1   0.0000000E+00   0.0000000E+00  -1.6534911E-01
   2   0.0000000E+00   1.0950442E-17   1.6534911E-01
   3   0.0000000E+00  -4.2436222E-02  -5.4914520E-02
   4   0.0000000E+00   4.2436222E-02   5.4914520E-02
   5   0.0000000E+00  -4.2436222E-02   5.4914520E-02
   6   0.0000000E+00   4.2436222E-02  -5.4914520E-02

