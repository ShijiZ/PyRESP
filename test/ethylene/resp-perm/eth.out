
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
 test for ethylene
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 1   iqopt       = 1
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00050000
 ipol        = 5   igdm        = 1
 exc12       = 0   exc13       = 0
 ipermdip    = 1   pwt         = 0.00050000
 virtual     = 0

 --------------------
 Molecule Information
 --------------------
 Single-molecule run

 Molecule 1 weight:     1.000 
 Molecule 1 name: ethylene
 Total charge (ich):-99
 Number of centers:  6
 Number of permanent dipoles: 10
    1    6    0    0    0    2
    2    6    1    1    2    2
    3    1    0    0
    4    1    3    7
    5    1    3    7
    6    1    3    7

 -----------------------------
 Atom/Dipole Ivary Information
 -----------------------------
    6    0    0    0    2
    6    1    1    2    2
    1    0    0
    1    3    7
    1    3    7
    1    3    7

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =    6
 Total number of permanent dipoles      =   10
 Weight factor on initial charge restraints=  0.000500
 Weight factor on initial dipole restraints=  0.000500
 There are  0 charge constraints

 Read polarizability and radii information from:
 ../../../polarizability/pGM-pol-2016-09-01
 Assume the units are in a.u.

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =     6
 total number of esp points =   452

 center       X               Y               Z
    1   0.0000000E+00   0.0000000E+00   1.2441670E+00
    2   0.0000000E+00   0.0000000E+00  -1.2441670E+00
    3   0.0000000E+00   1.7277940E+00   2.3159880E+00
    4   0.0000000E+00  -1.7277940E+00  -2.3159880E+00
    5   0.0000000E+00   1.7277940E+00  -2.3159880E+00
    6   0.0000000E+00  -1.7277940E+00   2.3159880E+00


 ------------------------
 Optimization Information
 ------------------------
 iqopt=1, all q0 and p0 values will be set to 0

 Number of unique UNfrozen charge centers =     2
 Number of unique UNfrozen permanent dipoles =     3

 Non-linear optimization requested.
  qchnge =    1.0789115490E-01
  qchnge =    2.4028969490E-02
  qchnge =    8.2967406685E-03
  qchnge =    3.4336824341E-03
  qchnge =    1.5061510196E-03
  qchnge =    6.7530563161E-04
  qchnge =    3.0533102090E-04
  qchnge =    1.3844782515E-04
  qchnge =    6.2815572660E-05
  qchnge =    2.8492842763E-05
  qchnge =    1.2917176507E-05
  qchnge =    5.8525320252E-06
  qchnge =    2.6502405814E-06
  qchnge =    1.1995686721E-06
  qchnge =    5.4274642757E-07

 Convergence in   14 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1: ethylene

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         6        0.000000     -0.240050      0       0.001923
    2         6        0.000000     -0.240050      1       0.001923
    3         1        0.000000      0.120521      0       0.000000
    4         1        0.000000      0.120521      3       0.000000
    5         1        0.000000      0.120521      3       0.000000
    6         1        0.000000      0.120521      3       0.000000

 Sum over the calculated charges:      0.002

       Permanent Dipoles Before & After Optimization
 dip.no   element.no   p(init)        p(opt)     ivary    d(rstr)/dp
    1         6        0.000000     -0.202861      0       0.002211
    2         6        0.000000      0.053659      0       0.004406
    3         6        0.000000      0.053659      2       0.004406
    4         6        0.000000     -0.202861      1       0.002211
    5         6        0.000000      0.053659      2       0.004406
    6         6        0.000000      0.053659      2       0.004406
    7         1        0.000000     -0.117910      0       0.000000
    8         1        0.000000     -0.117910      7       0.000000
    9         1        0.000000     -0.117910      7       0.000000
   10         1        0.000000     -0.117910      7       0.000000

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          2.0365103E-02
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   3.1378329E-04
 The root-mean-squared error (RMSE = sqrt(RSS/N))               8.3319315E-04
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         1.2412853E-01

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1        0.00000    0.00000   -0.00000   -0.00000

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   -2.09943
        Y    0.00000    0.91017
        Z    0.00000    0.00000    1.18925

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X    1.18925
        Y    0.00000    0.91017
        Z    0.00000    0.00000   -2.09943